<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>960</width>
      <height>720</height>
     </size>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="loadImage">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>이미지 로드</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convertImage">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>이미지 변환</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>721</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewOriginal">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>320</width>
        <height>320</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelImage">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>원본 이미지</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResult">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>190</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>변환 결과</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="logBrowser">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>570</y>
        <width>721</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelLog">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>550</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>상세 로그</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewResult">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>210</y>
        <width>320</width>
        <height>320</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelImage_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>진행률</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="browseOriginalDir">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>찾아보기</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirOriginal">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelOriginalDir">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>원본 이미지 폴더 경로:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResultDir">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>결과 이미지 폴더 경로:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirResult">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browseResultDir">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>찾아보기</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImageWidth">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>결과 이미지 크기(가로):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imageWidth">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelImageHeight">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>결과 이미지 크기(세로):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imageHeight">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="saveSetting">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>설정 저장</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>과정 표시</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImageHeight_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>변환 과정 시각화:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewMain"/>
    <addaction name="actionViewPreferences"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <action name="actionViewMain">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
  <action name="actionViewPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
